votey/slack.py

```diff
@@ -153,9 +157,11 @@ def handle_button_interaction(req: JSON) -> str:
         db.session.commit()
 
         position = 0
-        for button in attachments[1].get('actions'):
-            if int(button.get('value')) == option.id:
-                position = int(button.get('id'))
+        for attachment in attachments:
+            if attachment.get('actions'):
+                for button in attachment.get('actions'):
+                    if int(button.get('value')) == option.id:
+                        position = int(button.get('id'))
```

Comment:
oof, this is fine for now but we should BOLO for other slack apps that deal with this and see if we can avoid this guy lol